ipfs/flags.go

```diff
@@ -0,0 +1,25 @@
+package ipfs
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// AddFlags sets IPFS related flags to the given command.
+func AddFlags(cmd *cobra.Command, cfg *Config) {
+	cmd.Flags().String(
+		"ipfs.repo-path",
+		cfg.RepoPath,
+		"custom IPFS repository path. Defaults to `.{RootDir}/ipfs`",
+	)
+	cmd.Flags().Bool(
+		"ipfs.serve-api",
+		cfg.ServeAPI,
+		"set this to expose IPFS API(useful for debugging)",
+	)
+	cmd.Flags().BoolVar(
+		&EmbeddedInit,
+		"ipfs.init",
+		false,
+		"set this to initialize repository for embedded IPFS node. Flag is ignored if repo is already initialized",
```

Comment:
Out of curiosity what was the reasoning for using `false` as the default?

Reply:
It would be `false` anyway, even if I would have used the value from the config, as I did for other fields. The reason it does not get values from the config is that it does not make sense to keep `ipfs.init` in the config, as it only used once and won't have any effect if persistent.